crates/biome_js_analyze/src/lint/style/use_naming_convention.rs

```diff
@@ -305,6 +305,8 @@ declare_rule! {
     ///
     /// ### conventions
     ///
+    /// **This option will be available in Biome 1.8.**
```

Comment:
I just suggest just adding "Since (1.8.0)" next to the new option, otherwise you'll have to change this text after the release, which isn't productive.